CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Change history for stripes-core
 
 ## 9.1.0 IN PROGRESS
+* App context menu does not fully close after switching from "MARC Authority" app to "Inventory". Fixes STCOR-680.
```

Comment:
When fixing a bug, ideally the commit subject and CHANGELOG entry describe what changed rather than what the problem was. e.g. "Close app-context menu when its wrapper is undefined, which it will be when switching among apps". Otherwise, we end up with a changelog that looks like "Fix bug with menu. Fix bug with icon. Fix bug with locale." which isn't really helpful.

Reply:
Thanks, I've fixed it.